### ISSUE TITLE
Accessibility Testing for [VSA Debt, Financial Status Report, Enhanced Financial Status Report]

### ISSUE THREAD

**Issue by OptionSelect** [department-of-veterans-affairs/va.gov-team#59390]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@josephrlee 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Found an issue with the custom delete button we created not having sufficient contrast ratio on hover. Was fixed by this [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59517).

### Who completed the axe scans?

@OptionSelect 

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/58161

Axe Deque testing ticket. No issues were found in the scans.

### How did axe testing go?

No questions at this time.

### Who completed the content zoom and reflow test?

@josephrlee 

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Testing went well. Found two issues. One was submitted as a [bug request to the DST](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1797). The other bug will be [fixed on our end](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59524). Once the dependants width issue is fixed, we can check off the 300% zoom, but the 400% zoom will still be an issue as it belongs to the DST. 

### Who completed the keyboard navigation test?

OptionSelect

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Went well, the form was entirely accessible via keyboard.

### Do you have any other results to share?

N/A

**Comment by OptionSelect** [comment]
Please let me, @Dr-Pongo, and or @josephrlee know if there's anything that I missed or mistook in this process!

**Comment by josephrlee** [comment]
This is all up to date now. The only issue that remains is a DST bug with the `radio option` being set as a fixed width. But I've [filed an issue for it](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1797), so a fix should be in the works.

**Comment by josephrlee** [comment]
@briandeconinck , we have completed the accessibility testing on our end and are going to close this out. Please let us know if you think there are any additional concerns or issues! Thank you!

**Comment by HeatherRienks** [comment]
Closing this ticket as all noted items have been completed.
